.infrastructure/10_task-role.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_iam_role" "task_role" {
+  name               = var.ecs_task_role
+  assume_role_policy = data.aws_iam_policy_document.task_role.json
+
+  tags = {
+    Name       = var.product_name
+    CostCenter = var.product_name
```

Comment:
Are this tags useful? I see that almost everything is tagged as `var.product_name`

Reply:
They are useful for the sole purpose of associating the cost of AWS resources with a specific cost center. If you don't use tagging resources on cost centers in AWS they are useless. Do I remove these tags?

Reply:
I see that almost everything is tagged as `var.product_name`, you can instead put them into [variables](https://github.com/pagopa/b2b-portals/blob/main/.infrastructure/98_variables.tf#L17-L21) or [main](https://github.com/pagopa/b2b-portals/blob/main/.infrastructure/00_main.tf#L16-L18) files.